### PR TITLE
[stable/metricbeat] Fix 'secrets.yaml' to make `ExtraSecrets`work.

### PR DESCRIPTION
**Pull request by imcheck**
#### What this PR does / why we need it:
This is related to [PR #14084](https://github.com/helm/charts/pull/14084).
I forgot that an access with `.Values` didn't work in `range` function so that I have to fix it.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/metricbeat/templates/secret.yaml [hunk 1]
```diff
@@ -72,8 +72,8 @@ kind: Secret
 metadata:
   name: {{ $secret.name }}
   labels:
-    app: {{ template "metricbeat.name" . }}
```
Comment (mumoshu) [inline stable/metricbeat/templates/secret.yaml:75]: Would you mind trying `{{ template "metricbeat.name" $ }}` instead? The `$` always point to the "root" template data which isn't mutated by `range`s.

Comment (imcheck) [reply stable/metricbeat/templates/secret.yaml:75]: Oh..Thank you for your good feedback :)

Comment (imcheck) [reply stable/metricbeat/templates/secret.yaml:75]: @mumoshu

**Pull request merged**